bundles/org.openhab.binding.powermax/src/main/resources/OH-INF/thing/ip.xml

```diff
@@ -17,8 +17,11 @@
 			<channel id="with_zones_bypassed" typeId="with_zones_bypassed"/>
 			<channel id="trouble" typeId="trouble"/>
 			<channel id="alert_in_memory" typeId="alert_in_memory"/>
+			<channel id="ringing" typeId="ringing"/>
 			<channel id="pgm_status" typeId="pgm_status"/>
 			<channel id="mode" typeId="mode"/>
+			<channel id="last_message" typeId="last_message"/>
```

Comment:
To be coherent with my suggestion: `<channel id="last_message_time" typeId="last_message_time"/>`